bot/university/university.py

```diff
@@ -0,0 +1,378 @@
+        `ufind`.
+
+    Returns:
+        discord.Embed: Embedded message which lists all of the staff members returned by `ufind`.
+    """
+    list_selection_emojis = SelectionEmoji.to_list()
+    description = "Sollte die gesuchte Person hier nicht aufgelistet sein, dann versuche bitte deine Suchanfrage zu " \
+                  "spezifizieren.\n" + constants.ZERO_WIDTH_SPACE
+
+    embed = discord.Embed(color=constants.EMBED_COLOR_SELECTION, title="Wähle einen Mitarbeiter aus:",
+                          description=description)
+
+    for index, person in enumerate(persons):
+        name = "{0} {1} {2}".format(list_selection_emojis[index], person.findtext("firstname"),
+                                    person.findtext("lastname"))
+        embed.add_field(name=name, inline=True, value=constants.ZERO_WIDTH_SPACE)
+
+    return embed
+
+
+def _parse_staff_xml(str_xml: str) -> Dict[str, Union[datetime, str, None]]:
+    """Method for parsing the staff member data returned by `ufind` in XML.
+
+    Parses and prepares the data about a staff member and returns it in a dictionary so it can be posted in an
+    embed later.
+
+    Args:
+        str_xml (str): String containing data about a staff member in XML.
+
+    Returns:
+        Dict[str, str]: A dictionary containing the wanted information about a staff member.
+    """
+    xml = ET.fromstring(str_xml)
+
+    str_title = _create_staff_embed_title(xml.find("title"), xml.findtext("firstname"), xml.findtext("lastname"))
+    str_contact = _create_staff_embed_contact(xml.attrib["id"], xml.find("contact"))
+    str_office_hours = _create_staff_embed_office_hours(xml.findtext("hours"))
+    str_weblinks = _create_staff_embed_weblinks(xml.attrib["id"], xml.findtext("url"), xml.findtext("ucris"))
+    str_assignments = _create_staff_embed_assignments(xml.find("assignments"))
+    str_teaching = _create_staff_embed_teaching(xml.attrib["id"], xml.find("teaching"))
+
+    dict_staff = {
+        "title": str_title,
+        "contact": str_contact,
+        "weblinks": str_weblinks,
+        "office_hours": str_office_hours,
+        "assignments": str_assignments,
+        "teaching": str_teaching,
+        "last_modified": datetime.strptime(xml.attrib["version"], "%Y-%m-%dT%H:%M:%S%z"),
+    }
+    return dict_staff
+
+
+def _create_staff_embed_title(title: Optional[ET.Element], firstname: Optional[str], lastname: Optional[str]) \
+        -> Optional[str]:
+    """Method for building the string used as a title for the staff embed.
+
+    Args:
+        title (Optional[ET.Element]): Element containing the XML data regarding a staff members title.
+        firstname (Optional[str]): The firstname of the staff member.
+        lastname (Optional[str]): The lastname of the staff member.
+
+    Returns:
+        Optional[str]: The title for the staff embed.
+    """
+    if title is None:
+        return None
```

Comment:
`return title` 
No need to generate use None

Reply:
It doesn't matter.
> There is only one copy of that object in a given Python interpreter session.
https://stackoverflow.com/a/19473207/11889326